setup.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""The setup script."""
+
+from setuptools import setup, find_packages
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read()
+
+requirements = []
+
+setup_requirements = ['pytest-runner', ]
+
+test_requirements = ['pytest', 'flake8']
+
+setup(
+    author="Jason Joyce",
+    author_email='fuzzball81@gmail.com',
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
+        'Natural Language :: English',
+        "Programming Language :: Python :: 2",
```

Comment:
Probably don't want python 2 listed here?

Reply:
Yeah; should be python :: 3